src/hapi/pipelines/utilities/operational_presence.py

```diff
@@ -0,0 +1,201 @@
+"""Functions specific to the operational presence theme."""
+import hxl
+
+from logging import getLogger
+from typing import Dict
+
+from sqlalchemy.orm import Session
+
+from hdx.scraper.base_scraper import BaseScraper
+from hapi.pipelines.database.db_operational_presence import DBOperationalPresence
+from hapi.pipelines.utilities.admins import (
+    Admins,
+    get_admin2_to_admin1_connector_code,
+)
+# from hapi.pipelines.utilities.metadata import Metadata
+from hapi.pipelines.utilities.org import Org
+from hapi.pipelines.utilities.orgtype import OrgType
+from hapi.pipelines.utilities.sector import Sector
+
+logger = getLogger(__name__)
+
+
+class OperationalPresence(BaseScraper):
+
+    def __init__(self, session: Session, datasetinfo: Dict):
```

Comment:
If you pass all the data info that you need here, e.g. sector, org_type, then you can save them as attributes and you don't have to pass them around (e.g. see what I did with population). And functions like _get_sector_info can become instance methods that use `self._sector` etc